meetings/2022-12-14.md

```diff
@@ -69,11 +94,55 @@
+  >Through the survey, we wanted to learn about the members who comprise the community, discover issues people might be facing within the community, and how to make the community stronger, more welcoming, and ultimately, larger.
+
+* SC: Results of survey published on this link.
+* SC: Is the survey collecting what we need? We could put together survey to collect information about where people are finding info, also how people are being notified about meetings etc.  I could create an issue on that and we can come up with survey questions and look at it in follow-up meetings. Any ideas on that?  The scheduling, whether meeting time is ok, or languages that user stories are available in, I'm just looking through the survey and thinking of how it applies to use cases.
+* WT: This is so broad, it's hard to pin something down.  You are just talking about a survey, not something already delimited in a way?
+* SC: In context we are talking bout, which use cases may be of interest to implementors.
+* SC: Let me pull this link up, I've been pushing this for a while, but then we put it aside. 
+* SC: https://github.com/solid/authorization-panel/blob/main/proposals/authorization-ucr/uc-survey.md is one survey to collect some data about proposed use cases, commitment to implement or already implementing. This is quite important before dividing into actual requirements in a spec. It is also good signal to the community whether a feature will actually be implemented - towards "adequate implementation experience" as per W3C Process.
+* SC: We did a similar survey in the W3C Social Web Working Group. [Issue #9 from earlier](https://github.com/solid/specification/issues/9) mentions this as well.
+* SC: Use case being proposed, anyone contributing can say yes it is worth doing or I'm kind of for it and can live without it, and other options. This is the template I was proposing our understanding of the use cases and how it maps to a requirement in a spec. 
+* WT: Seems like a good thing to do. Then which use cases, right? If we agree this is a good format, then which use cases do you want to provide to do the survey with. The given UCR there are a lot of use cases surveyed.
+* SC: This is one of the steps just before writing a line of the spec. Just because someone proposed a use case, doesn't mean there has to be a requirement from it. It's to get a signal from the community that there is a significant amount of parties interested in a requirement for this use case. 
+* PA: I see in Issue #9 one issue document per spec, I'm not sure this is the right granularity. For example, for SPARQL, specifications are focused on a particular use case might be transversal, might gut feeling is it might be more efficient to have use cases pointing to the specification they touch on, not necessarily to segregate use cases per specification.
```

Comment:
```suggestion
* PA: I see in Issue #9, one issue document per spec, I'm not sure this is the right granularity. For example, for SPARQL, specifications are focused on a particular use case, might be transversal. My gut feeling is it might be more efficient to have use cases pointing to the specification they touch on, not necessarily to segregate use cases per specification.
```